There was an exception
I am inside finally block
